src/Scripts/Ecs/Systems/FovSystem.gd

```diff
@@ -0,0 +1,46 @@
+extends "res://Scripts/Ecs/Core/System.gd"
+
+var _event_bus
+var _player = null
+
+func _init(event_bus):
+	_event_bus = event_bus
+	
+	_event_bus.connect("spawn_entity", self, "on_spawn_entity")
+	_event_bus.connect("move_entity", self, "on_move_entity")
+
+func on_spawn_entity(entity):
```

Comment:
We use this pattern of "hook into spawn_entity to find the player" a few places, I wonder if we should instead just pass the player through the system constructor or do something else? It seems a big ... eh, excessive.

Reply:
We can have a designated signal!

Sometimes I feel like I'm overusing signals.